charts/airflow/README.md

```diff
@@ -171,6 +171,7 @@ Chart Version → <br> Airflow Executor ↓ | `7.X.X` | `8.X.X` |
 `CeleryExecutor` | ✔️ | ✔️
 `KubernetesExecutor` | ⚠️️ <sub>[1]</sub> | ✔️
 `CeleryKubernetesExecutor` | ❌ | ✔️
+`LocalExecutor` | ❌ | ✔️
```

Comment:
We would need to add a new column for whatever version we added support for `LocalExecutor` in.